.github/workflows/build_and_deploy.yml

```diff
@@ -10,6 +10,7 @@ env:
   TURBO_VERSION: 1.10.9
   NODE_MAINTENANCE_VERSION: 16
   NODE_LTS_VERSION: 18.16.1
+  CARGO_PROFILE_RELEASE_LTO: 'true'
```

Comment:
Didn't we disable this for some reason? @kwonoj

Reply:
I moved it from the `Cargo.toml`

Reply:
I didn't touch lto related options, probably it's kdy's changes?